lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -43,6 +43,7 @@ def []=(str, value)
 end
 
 RSpec.configure do |config|
+  config.include LogStashHelper
```

Comment:
I tend to prefer the syntax where in the Helper include, you add also the extend call ... I find this way more clear than this one in here that basically need two calls, but is my experience, do we want to keep this logic within the module?